llap-server/src/java/org/apache/hadoop/hive/llap/io/decode/GenericColumnVectorProducer.java

```diff
@@ -220,6 +220,9 @@ public static List<TypeDescription> setTypeBuilderFromSchema(
         case TIMESTAMP:
           type.setKind(OrcProto.Type.Kind.TIMESTAMP);
           break;
+        case TIMESTAMP_INSTANT:
```

Comment:
Code is easy and seems legit. Just could you please write a test to cover the timestamp?

Reply:
Added a qtest.